class DataException(Exception):
    """Excepci√≥n personalizada para errores en la carga de datos."""
    def __init__(self, message):
        super().__init__(message)
